.pipelines/templates/obp-file-signing.yml

```diff
@@ -0,0 +1,152 @@
+parameters:
+  binPath: '$(ob_outputDirectory)'
+
+steps:
+- pwsh: |
+    $fullSymbolsFolder = '${{ parameters.binPath }}'
```

Comment:
We should manually validate Windows file signing

Reply:
I have validated it. For 1p and 3P signing.